portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -818,6 +818,313 @@ definition {
 		}
 	}
 
+	@description = "LPS-155288: Given categories filter using treeview component
+	And there is any category's checkbox selected
+	When user clicks 'Clear All' button
+	Then all checkboxes turns deselected"
+	@priority = "4"
+	test CheckAllCheckboxesTurnsDeselectedWithClearButtonAtTreeviewComponent {
+		property portal.acceptance = "true";
+
+		task ("Adding vocabulary and categories") {
+			JSONCategory.addVocabulary(
+				groupName = "Test Site Name",
+				title = "Vocabulary Name");
+
+			var categoryId = JSONCategory.addCategory(
+				groupName = "Test Site Name",
+				title = "Test 1",
+				vocabularyName = "Vocabulary Name");
+			var categoryId = JSONCategory.addCategory(
+				groupName = "Test Site Name",
+				title = "Test 2",
+				vocabularyName = "Vocabulary Name");
+			var categoryId = JSONCategory.addCategory(
+				groupName = "Test Site Name",
+				title = "Test 3",
+				vocabularyName = "Vocabulary Name");
+		}
+
+		task ("Adding Web Content") {
+			JSONWebcontent.addWebContent(
+				assetCategoryIds = "${categoryId}",
+				content = "WC WebContent Content",
+				groupName = "Test Site Name",
+				title = "WC WebContent Title");
+		}
+
+		task ("Open filter modal at Content Dashboard and filtering category") {
+			ContentDashboard.goToContentDashboard();
+
+			ContentDashboard.openFilterModal(filterOption = "Categories");
+
+			Portlet.expandTree();
+
+			Click(
+				key_nodeName = "Test 1",
+				locator1 = "Treeview#NODE_ITEM");
+
+			Click(
+				key_nodeName = "Test 2",
+				locator1 = "Treeview#NODE_ITEM");
+
+			Click(
+				key_nodeName = "Test 3",
+				locator1 = "Treeview#NODE_ITEM");
+		}
+
+		task ("Checking if the results bar appears") {
+			AssertTextPresent(
+				locator1 = "Treeview#FILTER_BY_RESULTS_BAR_MESSAGE",
+				value1 = "3 ");
+
+			AssertTextPresent(
+				locator1 = "Treeview#FILTER_BY_RESULTS_BAR_MESSAGE",
+				value1 = "Items Selected");
+		}
+
+		task ("Assert 'Clear All' disapears after clicking it") {
+			ClickNoError(
```

Comment:
Hey! What do you think about creating a reusable macro for `ClickNoError` and `AssertElementNotPresent` ?

Reply:
Hello Luiz! great idea! :)

I created the new macro, could you review it again? Thanks!